packages/mobile/src/app/ErrorMessages.ts

```diff
@@ -29,6 +29,7 @@ export enum ErrorMessages {
   INVITE_FAILED = 'inviteFailed',
   INVITE_OPEN_APP_FAILED = 'inviteOpenAppFailed',
   SEND_PAYMENT_FAILED = 'sendPaymentFailed',
+  PAYMENT_LIMIT_REACHED = 'paymentLimitReached',
```

Comment:
This string needs to map to an error message string in /mobile/locales/en-US/global.json and /mobile/locales/es-419/global.json

Reply:
Done

Reply:
I will work with Aaron on top of this PR to make sure we can pass in the data we need and the formatting looks good. He has a change that allows passing the data to the error component I think.